org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm

```diff
@@ -290,20 +320,22 @@ class Signature(metaclass = MetaSet):
         elif self.multiplicity == Multiplicity.One:
             ret = [set(i) for i in self.atoms]
         elif self.multiplicity == Multiplicity.Some:
-            ret = list(powerset(self.atoms) - set(set()))
+            ret = powerset(self.atoms) - set(set())
         else: # Set
-            ret = list(powerset(self.atoms))
-        random.shuffle(ret)
```

Comment:
Could you bring back the random.shuffle part back? It's needed for non-deterministic. Thank you!